src/alejandra/tests/cases/apply/out-tabs.nix

```diff
@@ -0,0 +1,114 @@
+[
```

Comment:
I'd recommend separating the indention and correctness tests as those are different features and their testing logic might change over time. Perhaps, the structure could be

```
tests/cases
  rules
    apply
      out.nix (uses default 2 spaces indent)
    list
    ...
  indentation
    apply
      out-tabs.nix (uses tabs)
      out-{custom} (optional, if we'd like to test different indentations)
    list
```

Reply:
I think you are right, it's also less verbose that way